bindings/nodejs/npm/android-arm64/README.md

```diff
@@ -0,0 +1,3 @@
+# `nodejs-android-arm64`
```

Comment:
are the arch specific folders automatically (re-)created, or do they need to be checked in? Otherwise we should gitignore them.

Reply:
I guess we need them, since they are create with `napi new`